pennylane/expval/__init__.py

```diff
@@ -50,5 +50,44 @@
     expval/cv
 """
 
-from .qubit import *
-from .cv import *
+from pennylane.qnode import QNode, QuantumFunctionError
+
+from .qubit import * #pylint: disable=unused-wildcard-import,wildcard-import
+from .cv import * #pylint: disable=unused-wildcard-import,wildcard-import
+
+from .cv import __all__ as _cv__all__
+from .qubit import __all__ as _qubit__all__
+
+class Identity(object): #pylint: disable=too-few-public-methods,function-redefined
+    r"""pennylane.expval.Identity(wires)
+    Expectation value of the identity observable :math:`\I`.
+
+    The expectation of this observable
+
+    .. math::
+        E[\I] = \text{Tr}(\I \rho)
+
+    corresponds to the trace of the quantum state, which in exact
+    simulators should always be equal to 1.
+
+    This is a placeholder for the Identity classes in expval.qubit and expval.cv
+    and instantiates the Identity appropriate for the respective device.
+    """
```

Comment:
Does this docstring appear in the docs at all? If not, we could shorted it to simply
```python
Expectation value of the identity observable :math:`\I`.
This is a placeholder for the Identity classes in expval.qubit and expval.cv
and instantiates the Identity appropriate for the respective device.
```

Reply:
Currently it doesn't appear. Just thought it would be good to document somewhere how the "overloading" of Identity works. Maybe it is good to also add a back reference (in the code, not in the docs) from the cv.Identity and qubit.Identity to this "placeholder" class?

Reply:
Yes, let's do that

Reply:
Done in the commit I will push in a second...